Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,70 @@
+# Requesting and Notifying Out Of Office time
+
+Someone who is Out Of Office means that they are not in their usual work place at their usual working hours. This can be when someone goes on holidays or when someone that usually works in the office is working from somewhere else.
+
+OOO time has to be requested and approved by your Line Manager and your Project Manager.
+
+There are different types of OOO and each one of them have different requirements for requesting and for being notified.
+
+## Holidays 🌴
+
+As per contract, we are entitled to 25 holiday days per year. You can book your holidays at your convience, you might just need to sync first with your Squad and the other iOS engineers to see if everybody is OK with it. 
+
+Ideally, you should give at least 2 weeks notice before taking any holiday but there are exceptions for emergency situations. If you want to book more than 5 consecutive days you ideally should give 4 weeks notice.
+
+### Steps to request
+
+1. Ask for approval from your PM - only if approved you can proceed to the next steps.
+2. Communicate it to your line manager and mention your PM approval
+3. Request in Bamboo
```

Comment:
If we're really asking the rest of the team for permission to go on holidays (see previous comment) then we're missing a step here